### PR TITLE
[WIP] dom: Replace activeElement with documentElement.activeElement

### PR DESCRIPTION
**Pull request by tadatuta**
DO NOT MERGE!
#744

### CODE REVIEW

**Comment by narqo** [comment]
Looks like something has been done here. Anything else? Should we merge it?

**Comment by tadatuta** [comment]
I've never tested the result. @sipayRT can you please check if it works?

**Comment by sipayRT** [comment]
@tadatuta @narqo works fine for me. Tested on IE8-10

**Comment by sipayRT** [comment]
wait a minute

**Comment by aristov** [comment]
@sipayRT ping

**Comment by tadatuta** [comment]
should be reproducible without modular system (need to find active element before DOMReady)

File: common.blocks/dom/dom.js [hunk 1]
```diff
@@ -32,8 +32,10 @@ provide(/** @exports */{
      * @returns {jQuery}
      */
     getFocused : function() {
-        // "Error: Unspecified error." in iframe in IE9
-        try { return $(document.activeElement); } catch(e) {}
+        var d = document,
```
Comment (aristov) [inline common.blocks/dom/dom.js:35]: `d` => `doc`

File: common.blocks/dom/dom.js [hunk 2]
```diff
@@ -32,8 +32,10 @@ provide(/** @exports */{
...
+            documentElement = d.documentElement;
```
Comment (aristov) [inline common.blocks/dom/dom.js:36]: `docElement`

**Pull request closed without merging**